Gemfile

```diff
@@ -1,21 +1,8 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-# Declare your gem's dependencies in paginative.gemspec.
+# Declare your gem's dependencies in dre.gemspec.
```

Comment:
I'm guessing this was a result of copying the gem file over from dre for testing different rails versions